source/memory.tex

```diff
@@ -5082,6 +5058,36 @@
 
 \rSec2[smartptr.adapt]{Smart pointer adaptors}
 
+\rSec3[smartptr.adapt.defns]{Definitions}
```

Comment:
ISO wants only a single section "Terms and Definition" (the one in clause 3); not sure whether this is too close for their taste.

Do we have other such "Definitions" sections elsewhere?